blobit-core/src/main/java/org/blobit/core/api/Configuration.java

```diff
@@ -58,6 +58,12 @@
     public static final String MAX_ENTRY_SIZE = "max.entry.size";
     public static final int MAX_ENTRY_SIZE_DEFAULT = 64 * 1024;
 
+    public static final String LEDGER_MIN_TTL = "ledger.min.ttl";
+    public static final long LEDGER_MIN_TTL_DEFAULT = 1000L * 60 * 60 * 24;
+
+    public static final String WRITER_MAX_TTL = "writer.max.ttl";
+    public static final long WRITER_MAX_TTL_DEFAULT = 1000L * 60 * 60 * 12;
+
```

Comment:
Why MIN_TTL is less of MAX_TTL? Maybe the name needs to be swapped (in my mind a minimum time to live should be always less of the maximum time to live)

Reply:
Reading again the change the 2 variable are different (one is for the writer and the other for the ledger), so maybe my comments is not really pertinent

Reply:
We are going to keep a writer alive for 12h
The GC task will drop ledgers that do not contain live blobs only after 24h

If you invert the values we will see:
- a writer still open after 12h up to 24h
- the GC deletes the ledger at 12h after the creation

I feel we can lower the values. Having big values protects us from clock drifts but it will set a minimal retention time that doesn't exist until now
So even if the application deletes all of the blobs in a ledger we won't reclaim space

Reply:
I think we can lower the time to 1h and 2h respectively. This should protect us from normal clock drifts (if your servers are drifting more than 1h you have bigger problems) and the retention is in the ballpark of the downstream applications.

Reply:
@aluccaroni  done